packages/ui-components/src/BalanceDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Balance } from '@polkadot/types';
+import React from 'react';
+
+import { DynamicSizeText } from './Shared.styles';
+
+export type BalanceDisplayProps = {
+  balance?: Balance,
+  fontSize?: string,
```

Comment:
proposal: move to standardized sizes that map to the css, as you did in the Margin component. e.g. fontSize: 'small' maps to 10px. for consistency and less guesswork. Can do this in a future PR

Reply:
I'll do this now